packer/README.md

```diff
@@ -0,0 +1,59 @@
+
+# CommandoVM & HashiCorp's Packer
+
+Welcome to the packer builder of CommandoVM - a fully automated installation of the customizable commandvm platform.
+
+## Requirements
+
+* a hypervisor ( below are current hypervisors )
+  * [Virtualbox](https://www.virtualbox.org/wiki/Downloads)
+* HashiCorp's [packer](https://www.packer.io/downloads)
+* [packer provisioner windows update](https://github.com/rgl/packer-provisioner-windows-update)
+* 60 GB Hard Drive
+* 2 GB RAM
+
+## Recommended
+
+* 80+ GB Hard Drive
+* 4+ GB RAM
+
+# Instructions
+
+## Standard install
+
+1. make sure you have all the [pre-reqs](#requirements).
+2. Download the zip from [https://github.com/fireeye/commando-vm](https://github.com/fireeye/commando-vm).
+3. Decompress the zip and cd into the [win10_1809 folder](/packer/win10_1809/).
+4. run the following command: `packer build packer build win10_1809_virtualbox_iso_to_finish.json`.
```

Comment:
duplicated words here, should be:

`packer build win10_1809_virtualbox_iso_to_finish.json`

Reply:
lol, my b it was late :upside_down_face: